docs/operations/hotplug_interfaces.md

```diff
@@ -100,6 +100,23 @@ You can now check the VMI status for the presence of this new interface:
 kubectl get vmi vmi-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
+### Migration based hotplug
+In case your cluster doesn't meet the [requirements](#requirements), it's possible to hotplug an interface by migrating the VM.
+The actual attachment won't take place immediately, and the new interface will be available in the guest once the migration is completed.
+
+#### Add new interface:
+```bash
+virtctl addinterface vmi-fedora --network-attachment-definition-name new-fancy-net --name dyniface1
+```
+At this point the new interface is added to the spec but will not be attached to the running VM. 
+
+#### [Migrate the VM:](./live_migration.md#initiate-live-migration)
```

Comment:
Please consider adding a sentence like "For more details, please see live migration" with a link, instead of using the heading as a link.

Reply:
I added reference to the migration docs.

Regarding the title I prefer to leave it imperative with a snippet for the migration command to make the flow clear (and not make the reader jump to other pages).